CMakeLists.txt

```diff
@@ -568,37 +568,32 @@ boost support or an external boost library.")
       ENDIF()
     ENDIF()
 
+    # Use slightly different error messages for bundled and non-bundled Eigen
     IF(EXISTS ${LIBMESH_ROOT}/include/Eigen)
       IF("${EIGEN3_ROOT}" STREQUAL "${LIBMESH_ROOT}/include/Eigen")
         # OK
+      ELSEIF(${IBAMR_USE_BUNDLED_EIGEN3})
```

Comment:
Can we add this check to the legacy build system as well?

Reply:
We don't need this check for the autotools-based build system. In particular, if we detect libMesh + bundled Eigen, then we will switch to libMesh's version of Eigen - i.e., its designed to not error-out but instead reconfigure itself. We can't do that with CMake since libMesh doesn't install the CMake module files.

Reply:
This patch is needed for people who use the copy of Eigen bundled with libMesh - if there aren't any other things that need to be changed here then could someone please approve and merge?